releases/v23_0_0.rst

```diff
@@ -0,0 +1,270 @@
+
+In ``lsst.pipe.tasks.PsfWcsSelectImagesTask``:
+
+The following configs and functionality have been moved to ``lsst.pipe.tasks.ComputeExposureSummaryStatsTask``:
+
+* ``starSelection``
+* ``starShape``
+* ``psfShape``
+* ``doLegacyStarSelectionComputation``: provides backwards compatibility.
+
+In ``lsst.pipe.tasks.InsertFakesTask``:
+
+* ``raColName``: Use ``ra_col`` instead.
+* ``decColName``: Use ``dec_col`` instead.
+* ``diskHLR``, ``aDisk``, ``bDisk``, ``paDisk``: Use ``disk_semimajor_col``, ``disk_axis_ratio_col``, and ``disk_pa_col`` to specify disk half-light ellipse.
+* ``nDisk``: Use ``disk_n`` instead.
+* ``bulgeHLR``, ``bulge_pa_col``, ``aBulge``, ``bBulge``, ``paBulge``: Use ``bulge_semimajor_col``, ``bulge_axis_ratio_col``, and  to specify disk half-light ellipse.
+* ``nBulge``: Use ``bulge_n`` instead.
+* ``magVar``: Use ``mag_col`` instead.
+* ``sourceSelectionColName``: Use ``select_col`` instead.
+
+
+.. _release-v23-0-0-deprecations:
+
+Deprecations
+============
+
+These packages/functions are deprecated and will not be available in the next major release.
+
+- :ref:`release-v23-0-0-deprecate-gen2`
+- :ref:`release-v23-0-0-deprecate-decamRawIngestTask`
+- :ref:`release-v23-0-0-deprecate-Filter`
+- :ref:`release-v23-0-0-deprecate-validate-drp`
+- :ref:`release-v23-0-0-deprecate-doApplyUberCal-in-forcedPhotCcd.py`
+- :ref:`release-v23-0-0-deprecate-configurations-in-isrTask.py`
+- :ref:`release-v23-0-0-deprecate-configurations-in-psfexStarSelector.py`
+
+.. _release-v23-0-0-deprecate-gen2:
+
+Deprecated Generation 2 Middleware
+----------------------------------
+Generation 2 middleware (Gen2) is no longer being developed and should not be used for new code.
+Gen2 infrastructure code (within e.g., :ref:`pipe_tasks <lsst.pipe.tasks>`, :ref:`pipe_base <lsst.pipe.base>`, :ref:`obs_base <lsst.obs.base>`, `daf_persistence <https://github.com:lsst/daf_persistence>`_, and obs packages) will no longer be maintained after January 1 2022 and may be removed at any point afterward.
+The CI package ``ci_hsc_gen2`` will continue to be run, and the tasks it checks will be maintained until we remove the Gen2 infrastructure code in 2022.
+Following this release, we will remove Gen2 infrastructure code and the commandline-task entry points, ``runDataRef`` to the Tasks.
+
+.. _release-v23-0-0-deprecate-decamRawIngestTask:
+
+Deprecated DecamRawIngestTask and MegaPrimeRawIngestTask
+-----------------------------------------------------------------
+MegaPrime and DECam no longer require a specialist Gen3 ingest task.
+Please use the default `~lsst.obs.base.RawIngestTask`.
+Both `~lsst.obs.decam.DecamRawIngestTask` and `~lsst.obs.cfht.MegaPrimeRawIngestTask` will be removed after v23.
+
+.. _release-v23-0-0-deprecate-Filter:
+
+Removal of ``lsst.afw.Filter``
+------------------------------
+The `lsst.afw.Filter` class has been replaced with :ref:`filterLabel <release-v22-0-0-filterLabel>`, is deprecated, and will be removed after this release.
```

Comment:
- Check the API name (not that it matters anymore). Is it actually `lsst.afw.image.Filter`?
- The link to `lsst.afw.Filter` doesn't work; you might want to use double backticks because I don't think that API is documented anyways (?)